osu.Game.Rulesets.Catch/Objects/Drawables/DrawableCatchHitObject.cs

```diff
@@ -19,10 +22,26 @@ public abstract class DrawableCatchHitObject : DrawableHitObject<CatchHitObject>
         protected DrawableCatchHitObject(CatchHitObject hitObject)
```

Comment:
should be annotated while we're at it

```suggestion
        protected DrawableCatchHitObject([CanBeNull] CatchHitObject hitObject)
```